cli/create_test.go

```diff
@@ -293,7 +293,7 @@ func TestCreateInvalidArgs(t *testing.T) {
 		},
 	}
 
-	testingImpl.CreateSandboxFunc = func(sandboxConfig vc.SandboxConfig) (vc.VCSandbox, error) {
+	testingImpl.CreateSandboxFunc = func(ctx context.Context, sandboxConfig vc.SandboxConfig) (vc.VCSandbox, error) {
```

Comment:
I wonder if wouldn't be better to define a global context variable instead of modify all the functions

Reply:
might be something like `traceContext` ..

Reply:
I did think about that (and in fact did use that technique in `virtcontainers/hack/virtc/main.go`), but I'd rather we use [the recommended approach](https://github.com/yurishkuro/opentracing-tutorial/tree/master/go/lesson02#propagate-the-in-process-context) and minimise globals where possible.

Reply:
make sense, thanks for the info